testdata/p4_16_samples/pna-dpdk-bvec_union.p4

```diff
@@ -1,5 +1,5 @@
 #include <core.p4>
-#include <pna.p4>
+#include <dpdk/pna.p4>
```

Comment:
For PSA, I thought we had something in the p4c-dpdk front end script that made it automatically look in the dpdk directory for the psa.p4 include file, so that P4 developers could simply write `#include <psa.p4>` and it would find it in the `p4include/dpdk` directory.  It would be nice to do the same for p4c-dpdk for PNA architecture programs.

Reply:
yes it only works for `p4c-dpdk` but not with `p4test`. When running tests(make check) every testfiles compiles with `p4test` and `p4c-dpdk`.